tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -129,3 +129,12 @@ Feature: create auth-app token
         }
       }
       """
+
+  @env-config
+  Scenario: user deletes auth-app token using token form list request
+    Given the config "AUTH_APP_ENABLE_IMPERSONATION" has been set to "true"
+    And user "Alice" has created app token with expiration time "72h" using the auth-app API
+    And the administrator has created app token for user "Alice" with expiration time "72h" using the auth-app API
+    And user "Alice" has created app token with expiration time "72h" using the auth-app CLI
+    When user "Alice" deletes all created auth-app token
```

Comment:
for future proof, i think we should specify which method used to delete

Reply:
there's no different way to delete app tokens.

Reply:
I think you should be more clear in your comment, @saw-jan. you mean this bug report is actual behavior https://github.com/owncloud/ocis/issues/10675 so there's only one way to delete app token